README.md

```diff
@@ -1,10 +1,40 @@
 # Cloud-Optimized Geospatial Formats Guide
 
-See the site [https://developmentseed.org/cloud-optimized-geospatial-formats-guide/](https://developmentseed.org/cloud-optimized-geospatial-formats-guide/)
+---
 
-This site is built using [Quarto](https://quarto.org/docs/get-started/)
-To preview the site locally, install quarto and run:
+**Guide**: <a href="https://tinyurl.com/cogeo-guide" target="_blank">tinyurl.com/cogeo-guide</a>
 
-```sh
-quarto preview
-```
+**Source Code**: <a href="https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide" target="_blank">https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide</a>
+
+---
+
+## Why This Guide Exists
+
+This guide aims to provide comprehensive documentation on cloud-optimized geospatial formats. Geospatial data is growing in volume and complexity, and this guide serves to highlight best practices for handling such data, especially in a cloud environment.
+
+For more details, visit the official site: [Cloud-Optimized Geospatial Formats Guide](https://developmentseed.org/cloud-optimized-geospatial-formats-guide/).
```

Comment:
Oh I was confused at first but this README isn't published to the website? I thought you were linking to itself.

Reply:
Yes, it's only on the repo!